### ISSUE TITLE
GCM appears to not work with the Java 9 runtime on OS X

### ISSUE THREAD

**Issue by dalemyers** [Microsoft/Git-Credential-Manager-for-Mac-and-Linux#74]
We've discovered that there are multiple issues when running GCM on OS X when using Java 9. Anyone on our team who is running version 9, as installed via Homebrew, was seeing an error whenever auth was needed for a git operation. The errors varied, but the most common was:

```
Fatal: java.lang.Error encountered.  Details:
   unexpected error
fatal: credential helper '!/Library/Java/JavaVirtualMachines/jdk-9.0.4.jdk/Contents/Home/bin/java -Ddebug=false -Djava.net.useSystemProxies=true -jar /usr/local/Cellar/git-credential-manager/2.0.3/libexec/git-credential-manager-2.0.3.jar' told us to quit
```

Removing Java 9 and installing Java 8 solved this issue for everyone, and no one is still having issues. 

However, we'd like to allow our team to upgrade to version 9, so is there an ETA on a fix for this? 

Thanks!

**Comment by jamesottaway** [comment]
See #71

**Comment by dalemyers** [comment]
Resolved with version 2.0.4. Thanks @novak !
